exec/cnex/lib/runtime_win32.c

```diff
@@ -0,0 +1,389 @@
+#include "runtime.h"
+
+#define COBJMACROS
+
+#include <iso646.h>
```

Comment:
Why include this header file if you're not going to use anything in it?

Reply:
Fixed.